scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -152,3 +154,42 @@ def __iter__(self):
 
     def get_generated_keys(self) -> List[str]:
         return [self._name]
+
+
+class AttackPointIteratorRepeat(AttackPointIterator):
+    """
+    Attack point iterator class that iterates 
+    over a configuration a repeat amount of times.
+    """
+
+    def __init__(self,
+                 operation: str,
+                 configuration: Dict,
+                 repetitions: int = 0) -> None:
+        """Initialize the repeated iterate if repetitions is not present
+          or set to 0 it will not repeat and if it set 
+          to a negative number it will do an infinite loop."""
```

Comment:
Where can I find documentation of different values of the `repetitions` parameter?